src/Grpc.AspNetCore/Grpc.AspNetCore.csproj

```diff
@@ -13,6 +13,12 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <!-- Avoid NU5128 warning. TFM in dependencies group of the nuspec must match lib/ref content. -->
+    <None Include="lib\**\*">
```

Comment:
Does this include a DLL in the package? I explicitly excluded this with `IncludeBuildOutput` on line 11.

Reply:
No it does not, it includes just the dummy file.

![image](https://user-images.githubusercontent.com/2030323/65468534-34080400-de19-11e9-855b-d14e5ac6eb34.png)